src/main/kotlin/uk/gov/justice/digital/hmpps/sentenceplan/entity/PractitionerEntity.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.sentenceplan.entity
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.transaction.annotation.Propagation
+import org.springframework.transaction.annotation.Transactional
+
+@Entity(name = "Practitioner")
+@Table(name = "practitioner")
+class PractitionerEntity(
+  @Id
+  @Column(name = "id")
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @JsonIgnore
+  val id: Long? = null,
+
+  @Column(name = "uuid")
+  val uuid: String,
```

Comment:
why `String` here?

Reply:
same as my comment in https://github.com/ministryofjustice/hmpps-sentence-plan/pull/81#discussion_r1758876072